src/pages/proposals/ActiveProposals.vue

```diff
@@ -108,9 +116,56 @@ export default {
       this.getSupply()
     }
   },
+  activated () {
+    if (this.dao) {
+      this.pagination.restart = true
+      this.resetPagination()
+    }
+  },
 
   methods: {
-    ...mapActions('ballots', ['getSupply'])
+    ...mapActions('ballots', ['getSupply']),
```

Comment:
is ballots getSupply already using the multi tenant contracts? 
![image](https://user-images.githubusercontent.com/8117939/153439711-b343582b-f058-400c-8156-12f8ca6ce1f6.png)